Calculator.Tests/Calculator.Tests/CalculatorTests.cs

```diff
@@ -0,0 +1,248 @@
+            string year = yearSelect.SelectedOption.Text;
+            string actualDate = day + "/" + month + "/" + year;
+            string expectedDate = DateTime.Today.ToString("d/MMMM/yyyy");
+            Assert.AreEqual(expectedDate, actualDate);
+
+        }
+
+        [Test]
+        public void TestSelectAnyStart()
+        { 
+            SelectElement daySelect = new SelectElement(element: browser.FindElement(By.Id("day")));
+            
+            SelectElement monthSelect = new SelectElement(element: browser.FindElement(By.Id("month")));
+            
+            SelectElement yearSelect = new SelectElement(element: browser.FindElement(By.Id("year")));
+            // 2 April 2022
+
+            daySelect.SelectByText("2");
+            monthSelect.SelectByText("April");
+            yearSelect.SelectByText("2022");
+
+            string actualDate = daySelect.SelectedOption.Text + " " + monthSelect.SelectedOption.Text + " " + yearSelect.SelectedOption.Text;
+            Assert.AreEqual("2 April 2022", actualDate);
+
+        }
+        [Test]
+        public void TestFinancialYearIsMandatoryField()
+        {
+            
+            // NEED HELP how to skip required field properly? 
+            bool d365 = browser.FindElement(By.Id("d365")).Selected;
+            bool d360 = browser.FindElement(By.Id("d360")).Selected;
+
+            Assert.IsTrue(d365 || d360);
+            Assert.IsFalse(d365 && d360);
+
+            
+
+        }
+
+        [Test]
+        // NEED HELP what is Obsolete, program asks to add it
+        
+        public void TestIncomeIsDisplayed()
+        {
+            // NEED HELP maybe I don't need to write here next 2 lines? This condition is is Setup, or not?
+            string Income = browser.FindElement(By.Id("income")).GetAttribute("value");
+            Assert.AreEqual("0.00", Income);
+
+        }
+
+        [Test]
+        public void TestInterestEarnedIsDisplayed()
+        {
+            string InterestEarned = browser.FindElement(By.Id("interest")).GetAttribute("value");
+            Assert.AreEqual("0.00", InterestEarned);
+
+        }
+        [Test]
+        
+        public void TestEndDateDataIsCorrect()
+
+        {
+            browser.FindElement(By.Id("term")).SendKeys("7");
+            DateTime expected = DateTime.Today.AddDays(+7);
+           
+            string EndDate = browser.FindElement(By.Id("endDate")).GetAttribute("value");
+            // NEED HELP it works for today only, I guess. How to make it work for any day of any year?
+            Assert.AreEqual(expected.ToString("dd/MM/yyyy"), EndDate);
+
+        }
+
+        [Test]
+        
+        public void TestInterestEarnedLayout()
+        {
+            // NEED HELP how to find aalign typo to make test fail?
+            string actual = browser.FindElement(By.XPath("/html/body/div/div/table/tbody/tr[7]/th[1]")).GetAttribute("aalign");
```

Comment:
The test should retrieve a valid attribute value.

```suggestion
            string actual = browser.FindElement(By.XPath("/html/body/div/div/table/tbody/tr[7]/th[1]")).GetAttribute("align");
```

Reply:
done